### PR TITLE
[O2B-1174] Add get all DPL Detectors endpoint

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- NA

Notable changes for developers:
- Add service method and endpoint serving DPL Detectors list

Changes made to the database:
- NA

### CODE REVIEW

**Comment by xsalonx** [comment]
To be merged after https://github.com/AliceO2Group/Bookkeeping/pull/1380

File: lib/server/controllers/dplDetectors.controller.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+const { updateExpressResponseFromNativeError } = require('../express/updateExpressResponseFromNativeError.js');
+const { getAllDplDetectors } = require('../services/dpl/getAllDplDetectors.js');
+
+/**
+ * Endpoint handler to list all dpl detectors
+ *
+ * @param {Object} _request the express request
```
Comment (martinboulais) [inline lib/server/controllers/dplDetectors.controller.js:19]: As we defined you can remove the params and simply put a description

File: lib/server/services/dpl/getAllDplDetectors.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ */
+
+const { DplDetectorRepository } = require('../../../database/repositories/index.js');
+
+/**
+ * Find and return all DPL detectors
+ *
+ * @return {Promise<SequelizeDplDetector[]>} the DPL detectors list
+ */
+exports.getAllDplDetectors = () => DplDetectorRepository.findAll();
```
Comment (martinboulais) [inline lib/server/services/dpl/getAllDplDetectors.js:21]: Please wrap this into a class, as for other services. Low level services may be skipped (and maybe should) but not the high level ones.

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/server/services/dpl/getAllDplDetectors.js (hunk 2 above)
Comment (xsalonx) [reply lib/server/services/dpl/getAllDplDetectors.js]: done

File: lib/server/controllers/dplDetectors.controller.js (hunk 1 above)
Comment (xsalonx) [reply lib/server/controllers/dplDetectors.controller.js]: applied

**Review by martinboulais — APPROVED** [review]

**Pull request merged**